spec/requests/api/v1/dynamic_field_groups_spec.rb

```diff
@@ -131,6 +136,17 @@
 
   describe 'PATCH /api/v1/dynamic_field_groups/:id' do
     let(:dynamic_field_group) { FactoryBot.create(:dynamic_field_group) }
+    let(:export_rule) do
+      FactoryBot.create(
+        :export_rule,
+        dynamic_field_group: dynamic_field_group,
+        field_export_profile: field_export_profile
+      )
+    end
+
+    before do
+      dynamic_field_group.export_rules << export_rule
```

Comment:
Is it necessary to add the export_rule to the dynamic_field_group's export_rules if you're running `create` on the export_rule above, and setting `dynamic_field_group: dynamic_field_group`?

Reply:
You're right. I should structure this test differently. Will fix.